src/utils/roofline_calc.py

```diff
@@ -44,13 +44,13 @@
 # SUPPORTED_DATATYPES table is based on datatype support in rocm-amdgpu-bench repository
 # Indicates which datatypes per gpu arch can be generated by the roofline binary
```

Comment:
Might want to add NB: this should be moved to a centrelized place later.
I believe it will not only for roofline:)